apps/web/app/[locale]/(app)/(auth)/reset-password/ResetPasswordForm.tsx

```diff
@@ -0,0 +1,162 @@
+'use client';
+
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useForm } from 'react-hook-form';
+import * as z from 'zod';
+import { Button } from '@/components/ui/button';
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from '@/components/ui/form';
+import { Input } from '@/components/ui/input';
+import { useRouter } from 'next/navigation';
+import { cn, getBaseUrl } from '@/lib/utils';
+import { useTranslations } from 'next-intl';
+import { useEffect, useState } from 'react';
+import { useToast } from '@/components/ui/use-toast';
+import Spinner from '@/components/spinner';
+import { useSupabase } from '@/lib/provider/supabase';
+
+interface ResetPasswordFormProps {
+  className?: string;
+}
+
+export const ResetPasswordForm: React.FC<ResetPasswordFormProps> = ({
+  className,
+}) => {
+  const router = useRouter();
+  const { supabase } = useSupabase();
```

Comment:
It could be sensible to display a toast (or redirect the user) if the auth session is missing. 

(or find a way to increase the session duration via Supabase 😄)

Reply:
Yep, fair point. I did both, so the user gets a toast notification that the link is expired and gets redirected to the `/forgot-password` page to request a new one. 🙌

Unfortunately, I couldn't find a way to increase the session duration.